### PR TITLE
New fix kevin

### PR DESCRIPTION
**Pull request by sorrento974**

### CODE REVIEW

File: client/src/main.jsx [hunk 1]
```diff
@@ -29,13 +29,10 @@ import EditProfile from "./pages/EditProfile";
 import EditPersonalInfo from "./components/ProfileForm";
 import Admin from "./pages/Admin";
 import Score from "./pages/Score";
-import AdminStreetArtPage from "./pages/AdminStreetArtPage";
```
Comment (Dinosaure7) [inline client/src/main.jsx]: Why do you delete this file import?
and the next.
Basically why delete all that is related to the street art page?

Comment (Dinosaure7) [reply client/src/main.jsx]: Saw with Kevin, all good.

**Review by Dinosaure7 — APPROVED** [review]

File: client/src/pages/Login.jsx [hunk 2]
```diff
@@ -34,6 +36,10 @@ function Login() {
 
         <button type="submit">Se connecter</button>
 
+        {actionData?.error && (
+          <div className="error-message">{actionData.error}</div>
```
Comment (Yoann-Fortin) [inline client/src/pages/Login.jsx]: A p tag will be better

File: client/src/styles/ZoomControl.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.custom-zoom-control .zoom-in {
+  padding-left: 0.15rem;
+}
+
+.custom-zoom-control button:hover {
+  color: var(--third-color);
+}
+@media screen and (min-width: 380px) and (max-width: 410px) {
```
Comment (Yoann-Fortin) [inline client/src/styles/ZoomControl.css]: The css is cascaded, and working in mobile first, the min-width is not necessary.

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**